src/components/ContactDetails/ContactDetailsAvatar.vue

```diff
@@ -26,7 +26,7 @@
 		<div class="contact-header-avatar__wrapper">
 			<div class="contact-header-avatar__background" @click="toggleModal" />
 			<div v-if="contact.photo"
-				:style="{ 'backgroundImage': `url(${contact.photoUrl})` }"
+				:style="{ 'backgroundImage': `url(${contact.photoUrl})`, 'backgroundColor': '#fff' }"
```

Comment:
Shall we use default background variable here @jancborchardt ?

Reply:
No, as said above at https://github.com/nextcloud/contacts/pull/1415#discussion_r368224620
> Just use `#fff` here, since it looks weird with the color still shining through. :) Also we don’t need to theme it because with dark mode, the backgrounds of these images should still be white.

Reply:
Oups, sorry! Missed this! :see_no_evil: